src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -0,0 +1,155 @@
+        self.assertLessEqual(transmit_expire_time_1, grant_expire_time_1)
+    del request
+    return response
+   
+  @winnforum_testcase
+  def test_WINNF_FT_S_HBT_9(self):
+    """ Array request, of three cbsds,SAS terminates the Grant for the one
+    that impacts incumbent activity.
+
+    successful heartbeat for the grants of two cbsds that don't impact the incumbent
+    and failed with responseCode 500 for the third one.
+    """
+    # STEP 1
+    # register three devices 
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_d.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    devices = [device_d, device_e, device_c]
+    for device in devices:
+        # inject FCC IDs
+        self._sas_admin.InjectFccId({'fccId': device['fccId']})
+    request = {'registrationRequest': devices}
+    response = self._sas.Registration(request)['registrationResponse']
+    cbsd_ids = []
+    for resp in response:
+        cbsd_ids.append(resp['cbsdId'])
+    cbsd_ids.sort(key=lambda cbsdId: cbsdId)
+    del request, response    
+    # create and send grant requests
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c['cbsdId'] = cbsd_ids[0]
+    grant_d = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_d['cbsdId'] = cbsd_ids[1]
+    grant_d['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_d['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    grant_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_e['cbsdId'] = cbsd_ids[2]
+    grant_e['operationParam']['operationFrequencyRange']['lowFrequency'] = 3560
+    grant_e['operationParam']['operationFrequencyRange']['highFrequency'] = 3570
+    request = {'grantRequest': [grant_c, grant_d, grant_e]}
+    # send grant request and get response
+    response = self._sas.Grant(request)['grantResponse']
+    self.assertEqual(len(response), 3)       
+    grant_ids = (response[0]['grantId'], response[1]['grantId'], response[2]['grantId'])
+    del request, response
+    # First Heartbeat
+    first_HB_response = self.sendAndAssertSuccessfulHeartBeat(cbsd_ids, grant_ids, [None, None, None])
+    transmitExpireTime = (first_HB_response[0]['transmitExpireTime'], first_HB_response[1]['transmitExpireTime'], first_HB_response[2]['transmitExpireTime'])
+    # STEP 2
+    # load and inject FSS data
+    fss_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_c.json')))
+    # Inject Incumbent Activity with Overlapping Frequency of CBSD
+    fss_c['deploymentParam']['operationParam']['operationFrequencyRange']['lowFrequency'] = \
```

Comment:
Isn't this supposed to overlap with the third CBSD's grant, not the second?

Also it probably makes more sense to set this to 3650-4200 which is more consistent with the FSS data we expect (and you can do this in the file below).

Reply:
thank you, I modifies it to be on 3650-4200 MHz.